db/knex_init_db.js

```diff
@@ -5,10 +5,11 @@ const { log } = require("../src/util");
  * ⚠️⚠️⚠️⚠️⚠️⚠️ DO NOT ADD ANYTHING HERE!
  * IF YOU NEED TO ADD FIELDS, ADD IT TO ./db/knex_migrations
  * See ./db/knex_migrations/README.md for more information
+ * @param {string} dbType "mariadb" or "postgres"
```

Comment:
The type of this parameter should be  `"mariadb"|"postgres"` given your comment, please change the type-definition and add a comment ^^